pipeline_dp/private_spark.py

```diff
@@ -0,0 +1,52 @@
+from pyspark import RDD
+from typing import Callable
+
+import pipeline_dp
+from pipeline_dp import aggregate_params, budget_accounting
+
+
+class PrivateRDD:
+    """A Spark RDD counterpart in PipelineDP guaranteeing that only anonymized data
```

Comment:
I like the text of this comment.

Nit: the [format](https://google.github.io/styleguide/pyguide.html#384-classes) of class comments according to styleguide
```

 """Summary of class here.

    Longer class information....
    Longer class information....

    Attributes:
```

I'd suggest something like:

  """ A Spark RDD counterpart.

 PrivateRDD guarantees that only anonymized data ...